mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -355,6 +382,7 @@ object ALS extends Logging {
    * Rating class for better code readability.
    */
   @DeveloperApi
+  @Since("1.3.0")
   case class Rating[@specialized(Int, Long) ID](user: ID, item: ID, rating: Float)
```

Comment:
Please annotate case class constructor parameters, [for example](https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/mllib/recommendation/ALS.scala#L32)